tests/test_search.py

```diff
@@ -102,7 +102,7 @@ def test_beam_search():
     results, mask, costs = search.search(
         {inputs: input_vals}, 0, 3 * length, as_arrays=True)
     # Just check sum
-    assert results.sum() == 3160
+    assert results.sum() == 1116
```

Comment:
That's because biases are initialized first in MLP, before weights. Which means that `use_bias=False` changes the initial random weights. This can not be called a bug, but I guess the other way round would make more sense.